gcb/rbe/configs/path/cc/BUILD

```diff
@@ -0,0 +1,151 @@
+
+# This becomes the BUILD file for @local_config_cc// under non-BSD unixes.
+
+package(default_visibility = ["//visibility:public"])
+
+load(":cc_toolchain_config.bzl", "cc_toolchain_config")
+load(":armeabi_cc_toolchain_config.bzl", "armeabi_cc_toolchain_config")
+load("@rules_cc//cc:defs.bzl", "cc_toolchain", "cc_toolchain_suite")
+
+licenses(["notice"])  # Apache 2.0
+
+cc_library(
+    name = "malloc",
+)
+
+filegroup(
+    name = "empty",
+    srcs = [],
+)
+
+filegroup(
+    name = "cc_wrapper",
+    srcs = ["cc_wrapper.sh"],
+)
+
+filegroup(
+    name = "compiler_deps",
+    srcs = glob(["extra_tools/**"], allow_empty = True) + [":builtin_include_directory_paths"],
+)
+
+# This is the entry point for --crosstool_top.  Toolchains are found
+# by lopping off the name of --crosstool_top and searching for
+# the "${CPU}" entry in the toolchains attribute.
+cc_toolchain_suite(
+    name = "toolchain",
+    toolchains = {
+        "k8|clang": ":cc-compiler-k8",
+        "k8": ":cc-compiler-k8",
+        "armeabi-v7a|compiler": ":cc-compiler-armeabi-v7a",
+        "armeabi-v7a": ":cc-compiler-armeabi-v7a",
+    },
+)
+
+cc_toolchain(
+    name = "cc-compiler-k8",
+    toolchain_identifier = "linux_gnu_x86",
+    toolchain_config = ":linux_gnu_x86",
+    all_files = ":compiler_deps",
+    ar_files = ":compiler_deps",
+    as_files = ":compiler_deps",
+    compiler_files = ":compiler_deps",
+    dwp_files = ":empty",
+    linker_files = ":compiler_deps",
+    objcopy_files = ":empty",
+    strip_files = ":empty",
+    supports_param_files = 1,
+    module_map = ":module.modulemap",
+)
+
+cc_toolchain_config(
+    name = "linux_gnu_x86",
+    cpu = "k8",
+    compiler = "clang",
+    toolchain_identifier = "linux_gnu_x86",
+    host_system_name = "i686-unknown-linux-gnu",
+    target_system_name = "x86_64-unknown-linux-gnu",
+    target_libc = "glibc_2.19",
+    abi_version = "clang",
+    abi_libc_version = "glibc_2.19",
+    cxx_builtin_include_directories = ["/usr/local/include",
+    "/usr/lib/llvm-10/lib/clang/10.0.0/include",
+    "/usr/include/x86_64-linux-gnu",
+    "/usr/include",
+    "/usr/lib/llvm-10/lib/clang/10.0.0/share",
+    "/usr/include/c++/9",
+    "/usr/include/x86_64-linux-gnu/c++/9",
+    "/usr/include/c++/9/backward",
+    "/usr/lib/clang/10.0.0/include"],
+    tool_paths = {"ar": "/usr/bin/ar",
+        "ld": "/usr/bin/ld",
+        "llvm-cov": "None",
+        "cpp": "/usr/bin/cpp",
+        "gcc": "/usr/bin/clang",
+        "dwp": "/usr/bin/dwp",
+        "gcov": "/usr/bin/gcov",
+        "nm": "/usr/bin/nm",
+        "objcopy": "/usr/bin/objcopy",
+        "objdump": "/usr/bin/objdump",
+        "strip": "/usr/bin/strip"},
+    compile_flags = ["-U_FORTIFY_SOURCE",
+    "-fstack-protector",
+    "-Wall",
+    "-Wthread-safety",
+    "-Wself-assign",
+    "-fcolor-diagnostics",
+    "-fno-omit-frame-pointer"],
+    opt_compile_flags = ["-g0",
+    "-O2",
+    "-D_FORTIFY_SOURCE=1",
+    "-DNDEBUG",
```

Comment:
Should we be turning assertions off? Usually I like to have them on at runtime, but if this is a file we got from elsewhere, this is fine.

Reply:
Auto generated. I will leave this as is for now.